README.md

```diff
@@ -1,229 +1,67 @@
-
-### Getting started
-
-AzCopy is self-documenting. To list the available commands, run:
-```
-./azcopy -h
-```
-
-To view the help page and examples, run:
-```
-./azcopy <cmd> -h
-```
-
-### Simple command-line syntax
-```
-# The general syntax
-./azcopy <cmd> <arguments> --<flag-name>=<flag-value>
- 
- # Example:
-./azcopy cp <source path> <destination path> --<flag-name>=<flag-value>
-./azcopy cp "/path/to/local" "https://account.blob.core.windows.net/container?sastoken" --recursive=true
-./azcopy cp "/mnt/myfile.txt" "https://myaccount.blob.core.windows.net/mycontainer/myfile.txt?sv=2017-11-09&ss=bf&srt=co&sp=rwac&se=2018-11-16T03:59:09Z&st=2018-11-15T19:59:09Z&sip=10.102.166.17&spr=https,http&sig=k8xSm2K3crBbtNpfoxyvh9n%2BMjDTqRk2XpY8JYIAMaA%3D"
- ```
-
-To see more examples:
-```
-./azcopy cp -h
-```
+AzCopy v10 is a command-line utility that you can use to copy data to and from containers and file shares in Azure Storage accounts. AzCopy V10 presents easy-to-use commands that are optimized for performance.
 
-Each transfer operation will create a `Job` for AzCopy to act on. You can view the history of jobs using the following command:
-```
-./azcopy jobs list
-```
+## Features and capabilities
 
-The job logs and data are located under the $HOME/.azcopy directory on Linux, and %USERPROFILE%\.azcopy on Windows. You can clear the job data/logs if you wish after AzCopy completes the transfers.
+:white_check_mark: Use with storage accounts that have a hierarchical namespace (Azure Data Lake Storage Gen2).
 
-### Copy data to Azure storage
-The copy command can be used to transfer data from the source to the destination. The locat can be a:
-1. local path
-2. URL to Azure Blob/Virtual Directory/Container
-3. URL to Azure File/Directory/File Share
-```
-./azcopy <command> <source path> <destination path>
-```
-
-The following command will upload `1file.txt` to the Block Blob at `https://myaccount.blob.core.windows.net/mycontainer/1file.txt`.
-```
-./azcopy cp /data/1file.txt "https://myaccount.blob.core.windows.net/mycontainer/1file.txt?sastokenhere"
-```
+:white_check_mark: Create containers and file shares.
 
-The following command will upload all files under `directory1` recursively to the path at `https://myaccount.blob.core.windows.net/mycontainer/directory1`.
-```
-./azcopy cp /data/directory1 "https://myaccount.blob.core.windows.net/mycontainer/directory1?sastokenhere" --recursive=true
-```
+:white_check_mark: Upload files and folders.
 
-The following command will upload all files directly under `directory1` without recursing into sub-directories, to the path at `https://myaccount.blob.core.windows.net/mycontainer/directory1`.
-```
-./azcopy cp /data/directory1/* "https://myaccount.blob.core.windows.net/mycontainer/directory1?sastokenhere"
-```
+:white_check_mark: Download files and folders.
```

Comment:
minor: consider using "directory" instead of "folder"

Reply:
Checking on this one as far as our style guide.

Reply:
Ok - changed from "folder" to "directory".